vms/platformvm/cache_locked_utxos_chain_state_test.go

```diff
@@ -0,0 +1,367 @@
+package platformvm
+
+import (
+	"testing"
+
+	"github.com/chain4travel/caminogo/ids"
+	"github.com/chain4travel/caminogo/vms/components/avax"
+	"github.com/chain4travel/caminogo/vms/secp256k1fx"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSemanticVerifyLockInputs(t *testing.T) {
+	vm, _, _ := defaultVM()
+	vm.ctx.Lock.Lock()
+	defer func() {
+		if err := vm.Shutdown(); err != nil {
+			t.Fatal(err)
+		}
+		vm.ctx.Lock.Unlock()
+	}()
+
+	type args struct {
+		inputs []*avax.TransferableInput
+		bond   bool
+	}
+	tests := []struct {
```

Comment:
please add test for bonding deposited utxo and vice versa - it should pass

Reply:
Should i also add these tests in `TestUpdateUTXOs`?

Reply:
yep, I think so
Still looking into `TestUpdateUTXOs` tests